lms/templates/main.html

```diff
@@ -102,6 +102,7 @@
       self.course_about_sidebar_extra_file = 'theme-course_about-sidebar_extra.html'
       self.course_about_social_sharing_file = 'theme-course_about-social_sharing.html'
       self.course_info_extra_file = 'theme-course_info_extra.html'
+      self.course_info_extra_file_sneakpeek = 'theme-course_info_extra_sneakpeek.html'
```

Comment:
You need to assign a default value above, else this will break for non Stanford-style themes.